graphqlapi/src/main/java/io/stargate/graphql/schema/graphqlfirst/fetchers/deployed/UpdateFetcher.java

```diff
@@ -16,24 +16,25 @@
 package io.stargate.graphql.schema.graphqlfirst.fetchers.deployed;
 
 import graphql.schema.DataFetchingEnvironment;
+import graphql.schema.DataFetchingFieldSelectionSet;
 import io.stargate.auth.Scope;
 import io.stargate.auth.SourceAPI;
 import io.stargate.auth.TypedKeyValue;
 import io.stargate.auth.UnauthorizedException;
 import io.stargate.db.datastore.DataStore;
+import io.stargate.db.datastore.ResultSet;
 import io.stargate.db.query.BoundDMLQuery;
 import io.stargate.db.query.builder.AbstractBound;
 import io.stargate.db.query.builder.BuiltCondition;
 import io.stargate.db.query.builder.ValueModifier;
 import io.stargate.db.schema.Keyspace;
-import io.stargate.graphql.schema.graphqlfirst.processor.EntityModel;
-import io.stargate.graphql.schema.graphqlfirst.processor.FieldModel;
-import io.stargate.graphql.schema.graphqlfirst.processor.MappingModel;
-import io.stargate.graphql.schema.graphqlfirst.processor.UpdateModel;
+import io.stargate.graphql.schema.graphqlfirst.processor.*;
 import io.stargate.graphql.web.StargateGraphqlContext;
 import java.util.*;
+import java.util.function.Function;
+import java.util.function.Predicate;
 
-public class UpdateFetcher extends DeployedFetcher<Boolean> {
+public class UpdateFetcher extends DeployedFetcher<Map<String, Object>> {
```

Comment:
This needs to be `Object`, we still allow the mutation to return a simple `Boolean`.